packages/schematics/angular/migrations/update-12/update-angular-config.ts

```diff
@@ -19,6 +19,12 @@ export default function (): Rule {
       for (const [, options] of allTargetOptions(target)) {
         delete options.experimentalRollupPass;
         delete options.lazyModules;
+        delete options.i18nLocale;
```

Comment:
I think instead of updating this migration we should re-run https://github.com/angular/angular-cli/blob/master/packages/schematics/angular/migrations/update-9/update-i18n.ts as we are going to remove the options from the other builders as well.

Reply:
sounds good.